usermods/multi_relay/readme.md

```diff
@@ -47,6 +47,19 @@ or
 
 You can override the default maximum number of relays (which is 4) by defining MULTI_RELAY_MAX_RELAYS.
 
+Some settings can be defined (defaults) at compile time by setting the following defines:
+
+MULTI_RELAY_HA_DISCOVERY true or false
```

Comment:
These changes do not render correctly when viewed as a document (i.e. viewed on Github).